packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -168,15 +168,6 @@ async function main() {
         organization: true
       },
       jest: {
```

Comment:
Do we still want to leave an empty `jest` section?

Reply:
Does the section stay so `cdk-build` can differentiate `nodeunit` packages from `jest` ones? If that is the case, should we switch to a different marker? For example, a declared `devDependency` on `@types/nodeunit`?

Reply:
> Does the section stay so cdk-build can differentiate nodeunit packages from jest ones?

Correct.

> Should we switch to a different marker? For example, a declared devDependency on @types/nodeunit?

Why change the marker? I, in fact, like this marker. Using dependencies to determine this sounds like 'magical' side effects to behaviour that users will not realize.

> It's usually not a good practice to rely on empty objects as values.

Can you expand? Perhaps you mean javascript objects (over the wire or in memory) in general, but do you think this strictly applies to `package.json` as well?

> Can we enforce this with pkglint?

What's the specific enforcement you're looking for? In case you missed it, the empty body signifies that none of the global configuration should be overridden. Packages may choose to override one or more of this, such as coverage (example: `aws-ses-actions/package.json`).

Reply:
> It's usually not a good practice to rely on empty objects as values.

It's just not good form. Sometimes (semantically) empty objects (`{ }`) are considered like undefined or falsy values and it's a good practice when designing a data protocol to rely on an empty object as a signifier. How about when you don't want to specify any special options you can just say `"jest": true` or `"jest": "defaults"` or something like that.

Reply:
> Can we enforce this with pkglint?

Perhaps we can enforce that package specific configuration does not duplicate the defaults. That is, if I put a value there that is already the default, it will remove it.

Reply:
I mean changing markers due to the fact we should eventually get rid of all `nodeunit` usage, and that should be what requires "additional entries".